install_static_swift_modules/after/Pods/Headers/Private/CustomModuleMapPod/CustomModuleMapPod-library-iOS.modulemap

```diff
@@ -1 +1 @@
-/Users/segiddins/Development/Rainforest/CocoaPods/tmp/install_static_swift_modules/transformed/Pods/Target Support Files/CustomModuleMapPod-library-iOS/CustomModuleMapPod-library-iOS.modulemap
\ No newline at end of file
+/Users/jkap/Development/CocoaPods/tmp/install_static_swift_modules/transformed/Pods/Target Support Files/CustomModuleMapPod-library-iOS/CustomModuleMapPod-library-iOS.modulemap
```

Comment:
re: discussion on Slack, these don't need to change but eventually we should build something that makes these work independently of the developer's file system structure

Reply:
wait shit i thought i got all of these. hold on a sec